config/dev.json

```diff
@@ -26,7 +26,7 @@
     "cluster/reseller/customer/domain/project/cloud-apps/data-browser"
   ],
   "port": 8822,
-  "sizeLimit": 524288000,
+  "sizeLimit": 1000000000,
```

Comment:
The description says that the 1Gb sizeLimit is removed but it looks like it's still in use?